src/app/clusters/level-control/level-control.cpp

```diff
@@ -110,6 +110,8 @@ typedef struct
 
 static EmberAfLevelControlState stateTable[kLevelControlStateTableSize];
 
+static chip::System::Clock::Timestamp nextTimestamp;
```

Comment:
Using a global variable seems incorrect given that there can be multiple endpoints with LevelControl cluster.